strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -465,8 +463,7 @@ protected void doDeletePath(RepositoryPath repositoryPath,
     {
         Files.delete(repositoryPath);
 
-        Repository repository = repositoryPath.getFileSystem()
-                                              .getRepository();
```

Comment:
if you countning the line of codes then why do we need `{` on the next line? 
It's certainly just sarcasm :)
But do you mind if I will format code parts which I worked on like below? 
```
foo.bar()
   .do()
   .it();
```